ocaml/xapi/xapi_vm_lifecycle.ml

```diff
@@ -300,6 +300,14 @@ let check_vgpu ~__context ~op ~ref_str ~vgpus =
   | `checkpoint | `migrate_send -> Some (Api_errors.vm_has_vgpu, [ref_str])
   | _ -> None
 
+let check_vusb ~vmr ~op ~ref_str =
```

Comment:
This could be also written as:

```
match op with 
| `suspend 
| `snapshot 
| `migrate_send 
| `pool_migrate  when vmr.Db_actions.vM_VUSBs <> [] -> Some (Api_errors.vm_has_vusbs, [ref_str])
| _ -> None
```